src/__tests__/ActionCreators.test.js

```diff
@@ -0,0 +1,55 @@
+import * as actions from '../actions';
+import {
+  RECEIVE_LOCATION,
+  CHANGE_STATUS,
+  INPUT_CHANGE,
+} from '../constants';
+
+describe('actions', () => {
+  it('creates an action to receive a location', () => {
+    const date = new Date();
+    const name = 'Richmond, VA';
+    const aqi = 62;
+    const description = 'Fair air quality';
+    const color = 'rgb(181, 226, 30)';
+    const location = {
+      date,
+      name,
+      aqi,
+      description,
+      color,
+    };
+
+    const expectedAction = {
+      type: RECEIVE_LOCATION,
+      date,
+      name,
+      aqi,
+      description,
+      color,
+    };
```

Comment:
you could probably smoosh all this down a little bit with the spread operator and a single assignment to location but it's a detail